frontend/src/i18n/locales/de.yaml

```diff
@@ -18,7 +18,14 @@ general:
   leave-page-confirmation: >
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
```

Comment:
Not sure I like the ellipsis. I personally would put a space in front of it, but also in general, it always looks kind of passive aggressive to me. :shrug:

Reply:
```suggestion
  no-root-children: Noch keine Seiten ...
```